packages/admin-panel/src/pages/resources/MapOverlayGroupRelationsPage.jsx

```diff
@@ -22,13 +22,13 @@ const FIELDS = [
     },
   },
   {
-    Header: 'Child Id',
-    source: 'child_id',
+    Header: 'Child Code',
+    source: 'map_overlay.code',
     type: 'tooltip',
     editConfig: {
       optionsEndpoint: 'mapOverlays',
-      optionLabelKey: 'mapOverlay.id',
-      optionValueKey: 'mapOverlay.id',
+      optionLabelKey: 'mapOverlay.code',
+      optionValueKey: 'mapOverlay.code',
       canCreateNewOptions: true,
       sourceKey: 'child_id',
     },
```

Comment:
Less confident about this one. Works, but I don’t understand it from first principles, so please take an extra few seconds on this one 😅

Reply:
I think this should work, assuming you have tested it?

Reply:
The autofill in the **New map overlay group relations* modal works, but I’ll double check it actually creates a valid entry in the Postgres table

Reply:
Turns out there are a few complications here. [Will make the updates to the **Map overlay group relations** page in a separate PR](https://linear.app/bes/issue/RN-1197/update-table-columns#comment-6f40b56e)